system/services/presideObjects/PresideObjectService.cfc

```diff
@@ -1260,6 +1261,10 @@ component displayName="Preside Object Service" {
 			args.filterParams = { "#idField#" = arguments.id, _version_changed_fields = "%,#args.fieldName#,%" };
 			args.delete( "fieldName" );
 			args.delete( "id" );
+		} else if ( listLen( arguments.id ) GT 1 ){
```

Comment:
Can we remove this? This is an unrelated change. You can achieve the same thing with:

```
selectData( filter={ id=arrayOfIds } );
```

This could be a useful approach, but I'd rather make that a separate change and rather use an array than a string list to avoid ambiguity.

Reply:
@DominicWatson 
This is the line called and send configIds,
\system\handlers\admin\SysConfig.cfc : 193, Here if I implement the above changes means I've to change the dataManager and dataManagerService.cfc. Or else we've create a new service which we've to achieve this changes.
BTW Not sure whether we need to implement this change.